### ISSUE TITLE
No errors reported.  Unsure if extension is even working.

### ISSUE THREAD

**Issue by bumper314** [DavidAnson/markdownlint#476]
I'm using [VSCodium](https://vscodium.com/) 1.63.2 on macOS 10.14.6.  I've installed and enabled markdownlint 0.45.0 and even restarted VSCodium.

Here's my settings.json
```
{
    "workbench.colorTheme": "Default Light+",
    "markdownlint.config": {
        "default": true,
        "MD001": true,
        "MD003": { "style": "setext" },
        "no-hard-tabs": false
    }
}
```

I've opened a large .md file with many issue sure to violate the linting rules I have defined, and yet I don't see any linting errors in the PROBLEMS tab.  I don't use VSCode much, so I'm at a complete loss as to what to check.  Any help is appreciated.
